README.md

```diff
@@ -1,4 +1,30 @@
+
 # Databox Container Manager
 
 Databox container manager and dashboard user interface part of the databox platform 
-see [the main repository](https://github.com/me-box/databox) for more information.
+see [the main repository](https://github.com/me-box/databox) for more information. 
+
+For developing Databox core components- Databox exposes following functions:
```

Comment:
These are internal functions and are not exposed to developers. 

The container manager exposes an HTTP rest API on port 8989. 

GET:
/api/datasource/list
/api/installed/list
/api/:type/list (:type is app or driver or store)
/list-apps

POST:
/api/install 
/api/restart
/api/uninstall